app/_layouts/extension.html

```diff
@@ -216,13 +304,47 @@
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% else %}
 {% navtabs %}
 {% navtab Admin API %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Route:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, select **Routes** in the left navigation.
```

Comment:
Should avoid specific references to location (since it can be subject to change), such as "left navigation". Something like "From the Dashboard menu, click Routes and click View for the Route row."

Reply:
The location is subject to change, but I need a way to distinguish the left navigation from the top navigation since there are two options. The page is Dashboard, but the navigation remains for all pages, so it is not specific to the Dashboard - however for now it stays on the left side of the screen.